draft-ietf-jose-hpke-encrypt.md

```diff
@@ -282,21 +355,48 @@ Where:
 
 * BYTE(255): A separator byte (0xFF) used to delimit fields.
 
-* ASCII(content_encryption_alg): Identifies the algorithm with which the HPKE-encrypted key MUST be used. Its
-  value MUST match the "enc" (encryption algorithm) header parameter in the JOSE Header. This field provides JWE context information to the key derivation process and serves two purposes:
-
-  1. Ensures that derived key material is cryptographically domain-separated between the JWE HPKE integrated encryption and Key Encryption modes.
-  2. Ensures that the derived key is bound to the selected content encryption algorithm.
+* ASCII(content_encryption_alg): Identifies the content encryption algorithm
+  with which the HPKE-encrypted Content Encryption Key (CEK) MUST be used.
+  Its value MUST match the "enc" (encryption algorithm) header parameter value
+  in the JOSE Header.
+  This field provides JWE context information to the key derivation process,
+  which ensures that the derived key is bound to the selected content encryption algorithm.
 
 * BYTE(255): A separator byte (0xFF) used to delimit fields.
 
-* recipient_extra_info: An octet string containing additional context information that the application
-  includes in the key derivation via the HPKE `info` parameter. Mutually known private information (a concept also utilized in {{NIST.SP.800-56Ar3}}) MAY be used in this input parameter. If no additional context information is provided, this field MUST be empty.
+* recipient_extra_info: An octet string containing additional context information
+  that the application includes in the key derivation.
+  Mutually known private information (a concept also utilized in {{NIST.SP.800-56Ar3}}) MAY be used in this input parameter.
+  If no additional context information is provided, this field MUST be the empty octet sequence.
+Note that JWE HPKE integrated encryption does not require `Recipient_structure` because the JWE Protected Header and JWE AAD are already provided as HPKE aad, which binds these parameters to the ciphertext.
```

Comment:
```suggestion

Note that Integrated Encryption does not use the `Recipient_structure` because the JWE Protected Header and JWE AAD are included in the HPKE aad value, which binds these parameters to the ciphertext.

```